### ISSUE TITLE
Error when sending network command

### ISSUE THREAD

**Issue by nielsvn92** [SteinHeselmans/DucoBox#26]
Hi, 

I'm running your module on my Raspberry Pi Zero with a USB TTL Serial converter attached ([link](https://www.aliexpress.com/item/1pcs-CP2102-module-USB-to-TTL-serial-UART-STC-download-cable-PL2303-Super-Brush-line-upgrade/32694152202.html?spm=a2g0s.12269583.0.0.2fd3698fStJuP1)), and I'm getting the error `TypeError: unicode strings are not supported, please encode to bytes: '\r'` when running the command `ducobox -l debug -p /dev/ttyUSB0`. Using the same converter, I was able to run the Duco Network Tool and view all the network nodes. But that was on my laptop though.

I tried to change your code to do `self._serial.write(b'\r')` (notice the `b`), but then I only get an empty reply from the serial connection.

Can you help me?

```
$ ducobox -l debug -p /dev/ttyUSB0

Welcome to Duco Interface
Opened serial port /dev/ttyUSB0
Searching network...
Serial command:
network
Traceback (most recent call last):
  File "/usr/local/bin/ducobox", line 11, in <module>
    sys.exit(main())
  File "/home/pi/.local/lib/python3.5/site-packages/duco/ducobox.py", line 849, in main
    sys.exit(ducobox_wrapper(sys.argv[1:]))
  File "/home/pi/.local/lib/python3.5/site-packages/duco/ducobox.py", line 834, in ducobox_wrapper
    itf.find_nodes()
  File "/home/pi/.local/lib/python3.5/site-packages/duco/ducobox.py", line 766, in find_nodes
    reply = self.execute_command(self.LIST_NETWORK_COMMAND)
  File "/home/pi/.local/lib/python3.5/site-packages/duco/ducobox.py", line 718, in execute_command
    self._serial.write('\r')
  File "/home/pi/.local/lib/python3.5/site-packages/serial/serialposix.py", line 532, in write
    d = to_bytes(data)
  File "/home/pi/.local/lib/python3.5/site-packages/serial/serialutil.py", line 63, in to_bytes
    raise TypeError('unicode strings are not supported, please encode to bytes: {!r}'.format(seq))
TypeError: unicode strings are not supported, please encode to bytes: '\r'
```

**Comment by SteinHeselmans** [comment]
hmm, weird that i didn't see this yet in the past. I was able to reproduce the problem, and fix my reproduction scenario. I however don't have a connection to my ducobox in order to test the solution.
could you try my solution in #27  ?

**Comment by nielsvn92** [comment]
Thank you for the quick fix! Unfortunately I won't be able to test it until Monday. I'll let you know when I tested it.

**Comment by nielsvn92** [comment]
This indeed fixed the error. Thanks!

**Comment by SteinHeselmans** [comment]
fixed in release 0.3.1 (available in a few moments when travis finishes the CI)
